packages/frontend/core/src/blocksuite/ai/_common/chat-actions-handle.ts

```diff
@@ -535,16 +592,14 @@
   },
 };
 
-const CommonActions: ChatAction[] = [REPLACE_SELECTION, INSERT_BELOW];
-
 export const PageEditorActions = [
-  ...CommonActions,
+  PAGE_INSERT_BELOW,
   CREATE_AS_DOC,
   SAVE_CHAT_TO_BLOCK_ACTION,
 ];
 
 export const EdgelessEditorActions = [
-  ...CommonActions,
+  EDGELESS_INSERT_BELOW,
   ADD_TO_EDGELESS_AS_NOTE,
```

Comment:
Save as doc?

Reply:
Rename Insert Below to

* `PAGE_INSERT`
* `EDGELESS_INSERT`